#
# Sensor Axes 
#
# custom sensor_axes_t
	bool accel_x  # 1 = ACTIVE || 0 = INACTIVE
	bool accel_y  # 1 = ACTIVE || 0 = INACTIVE
	bool accel_z  # 1 = ACTIVE || 0 = INACTIVE
	bool gyro_x  # 1 = ACTIVE || 0 = INACTIVE
	bool gyro_y  # 1 = ACTIVE || 0 = INACTIVE
	bool gyro_z  # 1 = ACTIVE || 0 = INACTIVE
	bool mag_x  # 1 = ACTIVE || 0 = INACTIVE
	bool mag_y  # 1 = ACTIVE || 0 = INACTIVE
	bool mag_z  # 1 = ACTIVE || 0 = INACTIVE
